src/pages/EditMonitor.vue

```diff
@@ -532,6 +551,19 @@ export default {
         addedNotification(id) {
             this.monitor.notificationIDList[id] = true;
         },
+
+        /**
+         * Is the friendly name required for the selected option.
+         * Returns true if it is else null
+         * @returns {null|true}
+         */
+        friendlyNameRequired() {
+            if (this.friendlyNameRequiredOptions.includes(this.monitor.type)) {
+                return true;
+            } else {
+                return null;
```

Comment:
It would be better to simply return false here

Reply:
Fixed